src/styles/style.css

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap");
+ 
+
+.card_page {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 0 16px;
+}
+
+.container__img {
+  align-self: center;
+  margin-top: 32px;
+  height: 134px;
+  width: 160px;
+  margin-bottom: 40px;
+}
+
+.container__description {
+  margin-bottom: 4px;
```

Comment:
you used margin-top on the previous selector 
![image](https://user-images.githubusercontent.com/87998640/221665871-bf0dfaa2-fc11-4b3e-a9f2-94930c23a5fa.png)

Reply:
Can't remove it, because when I remove it, I failed tests, unfortunately